modules/nf-core/tcoffee/irmsd/tests/main.nf.test

```diff
@@ -52,25 +52,12 @@ nextflow_process {
     test("seatoxin - compressed") {
 
         setup {
-
-            run("UNTAR") {
-                script "../../../untar/main.nf"
-                process {
-                    """
-                    input[0] = [ [ id:'test' ],
-                                 file("https://raw.githubusercontent.com/nf-core/test-datasets/multiplesequencealign/testdata/structures/seatoxin-ref.tar.gz", checkIfExists: true)
-                               ]
-
-                    """
-                }
-            }
-
             run("PIGZ_COMPRESS") {
                 script "../../../pigz/compress/main.nf"
                 process {
                     """
                     input[0] = [ [ id:'test' ],
-                                file("https://raw.githubusercontent.com/nf-core/test-datasets/multiplesequencealign/testdata/setoxin.ref", checkIfExists: true)
+                                file(params.modules_testdata_base_path + "../../multiplesequencealign/testdata/setoxin.ref", checkIfExists: true)
                                ]
 
                     """
```

Comment:
I think you can move the setup to one setup block in the beginning of the tests. If you want to use different data just use aliases.

Reply:
Hi, i am not sure what this means - do you have any examples? Sorry and thanks a lot for the help!

Reply:
You can move 
```
            run("PIGZ_COMPRESS") {
                script "../../../pigz/compress/main.nf"
                process {
                    """
                    input[0] = [ [ id:'test' ],
                                file(params.modules_testdata_base_path + "../../multiplesequencealign/testdata/setoxin.ref", checkIfExists: true)
                               ]

                    """
                }
            }
```
into the setup block that you have at the beginning of the file :). That makes it a bit cleaner overall.